tests/dunit/src/test/scala/io/snappydata/dunit/streaming/StreamingDUnitTest.scala

```diff
@@ -1,45 +0,0 @@
-package io.snappydata.dunit.streaming
```

Comment:
After looking at docs, git actually does not have any rename operation. It just detects rename automatically based on content where it was failing for this case. Vivek has moved this to AQP so will delete it in any case.